openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/LoaderSlaveMultiThread.java

```diff
@@ -55,13 +57,42 @@ public static void main(String args[])
 		try
 		{
 			(new LoaderSlaveMultiThread()).run(args[0],Integer.parseInt(args[1]));
+			
+			// Print debug information: on Linux or AIX, run ps v <pid> on the JVM to get the RSS from the SZ column
```

Comment:
Pls remove "from the SZ column".  Using the new open `v` there is no SZ column, but a RSS column.

Reply:
I've updated the comment. Thanks!

Reply:
Please merge it.